### PR TITLE
Add radialIndicator@1.3.1 w/ git auto-update

### LINKED ISSUE #9161: [Request] Add radialIndicator
**Library name:** radialIndicator
**Git repository url:**https://github.com/s-yadav/radialIndicator.git
**npm package url(optional):** 
**License(s):**MIT
**Official homepage:**http://ignitersworld.com/lab/radialIndicator.html
**Wanna say something? Leave message here:**
# 

Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by qwe19272375**
Pull request for issue: #9161 
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/s-yadav/radialIndicator.git
 * Official website (optional, not the repository): http://ignitersworld.com/lab/radialIndicator.html
 * NPM package url (optional):
 * License and its reference: MIT, [ref](https://github.com/s-yadav/radialIndicator/blob/master/MIT-LICENSE.txt)
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 12
   - Count of stars: 114
   - Count of forks: 41
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #9161, cc @s-yadav

### CODE REVIEW

**Comment by qwe19272375** [comment]
@pvnr0082t Please help me review this pull request, thank you.

**Comment by qwe19272375** [comment]
@pvnr0082t Do we need to minify `angular.radialIndicator.js`?

**Comment by pvnr0082t** [comment]
Of course.

**Comment by qwe19272375** [comment]
angular files added.

File: ajax/libs/radialIndicator/package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  ],
+  "homepage": "http://ignitersworld.com/lab/radialIndicator.html",
+  "autoupdate": {
+    "source": "git",
+    "target": "https://github.com/s-yadav/radialIndicator.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "?(angular.)radialIndicator*"
```
Comment (pvnr0082t) [inline ajax/libs/radialIndicator/package.json]: @qwe19272375 We can simply use `*radialIndicator*`.

**Comment by qwe19272375** [comment]
`?(angular.)radialIndicator*` -> `*radialIndicator*`
@pvnr0082t please help me review it, thanks.

**Comment by maruilian11** [comment]
@PeterDaveHello I think this PR is ok.

**Pull request merged**